PowerUp/app/src/main/java/powerup/systers/com/IntroFragment.java

```diff
@@ -0,0 +1,27 @@
+package powerup.systers.com;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class IntroFragment extends Fragment {
+
+    int fragmentResource;
+
+    public IntroFragment() {
+    }
+
+    public IntroFragment(int id) {
+        fragmentResource = id;
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(fragmentResource, container, false);
+        return view;
```

Comment:
You don't need the variable here. You could directly return `inflater.inflate(fragmentResource, container, false)`